octopoes/octopoes/models/ooi/dns/records.py

```diff
@@ -185,3 +185,43 @@ class DNSCAARecord(DNSRecord):
     # without interior spaces or (2) a quoted string.
     value: str
     _natural_key_attrs = ["hostname", "flags", "tag", "value"]
+
+
+class DNSGPOSRecord(DNSRecord):
+    # RFC 1712
+    object_type: Literal["DNSGPOSRecord"] = "DNSGPOSRecord"
+    dns_record_type: Literal["GPOS"] = "GPOS"
+
+    latitude: str | None = None
+    longitude: str | None = None
+    altitude: str | None = None
```

Comment:
Shouldn't these be float types? Using the correct type would help make querying these objects easier

Reply:
Yes